README.md

```diff
@@ -8,7 +8,7 @@
 
 **Diagram as Code**.
 
-Diagrams lets you draw the cloud system architecture **in Python code**. It was born for **prototyping** a new system architecture design without any design tools. You can also describe or visualize the existing system architecture as well. Diagrams currently supports four major providers: `AWS`, `Azure`, `GCP` and `Kubernetes`.
+Diagrams lets you draw the cloud system architecture **in Python code**. It was born for **prototyping** a new system architecture design without any design tools. You can also describe or visualize the existing system architecture as well. Diagrams currently supports five major providers: `AWS`, `Azure`, `GCP`, `Oracle Cloud` and `Kubernetes`.
```

Comment:
`alibabscloud` provider was added. You need to update it!

ref: https://github.com/mingrammer/diagrams/pull/19